main.py

```diff
@@ -464,35 +474,172 @@ async def advance_game_turn(
     return {"status": "success"}
 
 
-@app.post("/api/lobby/{game_id}/movs", response_model=SetCardsResponse)
-async def deal_card_mov(
+@app.websocket("/ws/lobby/{game_id}/cardMovs")
+async def card_mov(
```

Comment:
```suggestion
async def notify_movement_card_changes(
```
Asi es mas descriptivo